models/api-model.js

```diff
@@ -0,0 +1,5 @@
+const fs = require("fs/promises");
+
+exports.selectApiInfo = () => {
+  return Promise.resolve(require("../endpoints.json"));
+};
```

Comment:
This is fine, but we technically don't need a model here and could just require the json into the controller and send it from there 🙂

Reply:
Is this because it's a simple access to data stored in the json? I was seeing the json as a sort of database but I guess since it's all synchronous it's all simple stuff - if it was instead stored in the database with everything else would a model make more sense in that case?